src/tests/specs/collaboratorTest.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { browser } from 'protractor';
+import { PlannerPage } from '../page_objects/planner';
+import * as support from '../support';
+
+
+describe('Planner Collaborator Tests:', () => {
+  let planner: PlannerPage;
+  let planner1: PlannerPage;
+  let c = new support.Constants();
+
+  beforeAll(async () => {
+    await support.desktopTestSetup();
+    planner = new PlannerPage(browser.baseUrl);
+    await planner.openInBrowser();
+    // // This is necessary since the planner takes time to load on prod/prod-preview
+    await browser.sleep(5000);
```

Comment:
We don't need this. `planner.openInBrowser` will perform login and that's all we need.
You can remove line 16, 17.

Reply:
Ok Thanks. I'll remove line 16,17